Recaptcha/RecaptchaVerifier.php

```diff
@@ -39,11 +39,14 @@ public function __construct(ReCaptcha $reCaptcha, RequestStack $requestStack, $e
 
     /**
      * Verify reCaptcha response.
+     *
+     * @param $recaptchaValue
+     *
+     * @throws RecaptchaException
      */
-    public function verify()
+    public function verify($recaptchaValue)
```

Comment:
Sorry I don't understand, you mean BC for person who have extends RecaptchaVerifier ? IMO RecapatchaVerifier must be final class, no needs to be extended, so no BC.